extensions/amp-analytics/amp-analytics.md

```diff
@@ -328,12 +328,13 @@ The element visibility trigger can be configured for any AMP element or a docume
 
 Notice that selector can be used to only specify a single element, not a collection. The element can be either an [AMP extended  element](https://github.xom/ampproject/amphtml/blob/master/spec/amp-tag-addendum.md#amp-specific-tags) or a document root.
 
-The element visibility trigger waits for element's [`ini-load`](#initial-load-trigger) signal before matching the `visibilitySpec`.
+The element visibility trigger waits for the signal specified by the `waitFor` property in `visibilitySpec` before tracking element visibility. If `waitFor` is not specified, it waits for element's [`ini-load`](#initial-load-trigger) signal when selector is specified. See `waitFor` docs for more details.
 
 
 <strong><a id="visibility-spec"></a>Visibility Spec</strong>
 
 The `visibilitySpec` is a set of conditions and properties that can be applied to `visible` or `hidden` triggers to change when they fire. If multiple properties are specified, they must all be true in order for a request to fire. Configuration properties supported in `visibilitySpec` are:
+  - `waitFor` This property indicates that the visibility trigger should wait for a certain signal before track visibility. The supported values are 'none', 'ini-load' and 'render-start'. If `waitFor` is undefined, it is defaulted to [`ini-load`](#initial-load-trigger) when selector is specified, or to `none` otherwise.
```

Comment:
Should 'none', 'ini-load', and 'render-start' be styled as `none`, `ini-load`, and `render-start`?